### DISCUSSION TITLE
Add zsh and some plugins to jail shell

### DISCUSSION THREAD

**Discussion by BenRoe** [Jip-Hop/jailmaker#101] [Q&A] [answered]
Hello,
if i create a jail with a config file and wanna pre install zsh and some plugins to the jail shell.
Is it correct to do it after `initial_setup=#!/usr/bin/bash`?

**Comment by Jip-Hop** [comment, marked as answer]
Yes, that's correct. Ensure the next lines start with white space. You can follow this as an example:

https://github.com/Jip-Hop/jailmaker/blob/main/templates/docker/config#L32-L45
